CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Breaking changes
+
+- fix(k8sprocessor)!: deprecate clusterName setting [#578]
```

Comment:
If I see correctly, you are just removing `clusterName`, it is no longer used.
For me deprecation is when something can be used, it is not recommended to use it and it will be removed in the future.

Reply:
I'm keeping the setting available, with a warning if it's used, it simply doesn't do anything anymore. So users don't have to update their configuration immediately, but the behaviour does change.